addons/mail/models/mail_activity_plan_template.py

```diff
@@ -127,7 +127,7 @@ def _get_date_deadline(self, base_date=False):
             return base_date + delta
         return base_date - delta
 
-    def _determine_responsible(self, on_demand_responsible, applied_on_record):
+    def _determine_responsible(self, on_demand_responsible, user_field_name, applied_on_record):
```

Comment:
as it is limited to "on demande", maybe name could be more precise ?
```suggestion
    def _determine_responsible(self, on_demand_responsible, on_demand_responsible_fname, applied_on_record):
```